gen2-depth-driven-focus/main.py

```diff
@@ -59,11 +59,11 @@ def create_pipeline():
     controlIn.out.link(cam.inputControl)
 
     left = pipeline.createMonoCamera()
-    left.setResolution(dai.MonoCameraProperties.SensorResolution.THE_480_P)
+    left.setResolution(dai.MonoCameraProperties.SensorResolution.THE_400_P)
```

Comment:
this is actually tricky one, and I'm not sure how we will resolve it. Thing is that OAK-D-Lite has sensor resolution of 480P, while all other models have 800,720 and 400P.

Reply:
I believe there was a plan on adding some common options for sensor resolution, CC @alex-luxonis @themarpe

Reply:
IMO lets do OAK-D for now and once we pin down Lite interoperability in this usecase we can change